<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="2">

  <testsuite name="stepsI&amp;II"
    errors="1" failures="0" hostname="" id="" package="stepsI&amp;II" skipped="1"
    tests="2" time="0.01600" timestamp="Wed, 25 Jul 2018 03:46:19 GMT">
  
    <testcase name="Q6M-27 input &amp; submit" classname="stepsI&amp;II" time="0.01600" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Steps I&amp; II / Q6M-27 input &amp; submit]:

  TypeError: WeatherMan.waitForElementPresent(...).setValue(...).click(...).waitForElementPresent(...).verify.containsText(...).refresh is not a function
    at Object.Q6M-27 input &amp; submit (C:\Users\Maximus Prime\Desktop\QA-Mountain\WeatherManII\nightwatch\tests\stepsI&amp;II.js:55:118)
    at Module.call (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:62:34)
    at C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:70:29
    at _fulfilled (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at process._tickCallback (internal/process/next_tick.js:150:11)
    </system-err>
  

  
    
    <testcase
      name="Q6M-28 Examine Card Elements" classname="stepsI&amp;II">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
